src/components/Artists/ArtistsList.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import ArtistItem from './ArtistItem.jsx';
+import PropTypes from 'prop-types';
+
+const ArtistsList = ({ artists }) => {
+  const artistsListItems = artists.map(artist => (
+    <li key={artist.artistId}>
+      <Link to={`/artist/${artist.artistId}`}>
+        <ArtistItem {...artist} />
+      </Link>
+    </li>
+  ));
+
+  return (
+    <section>
```

Comment:
I dont think these sections are needed unless you want to style them separate from the ul